internal/program.go

```diff
@@ -271,6 +271,16 @@ func program(ctx context.Context, _ []string, viper *viper.Viper, settings *cli.
 			return err
 		}
 	}
-
+	// Step 7: Export artifacts to json
+	if importConfig.Export.Artifacts.Enabled {
```

Comment:
I cannot see where `importConf.Export.Artifacts.Folder` is used for anything. Can you please point in the right direction? It feels like you are just writing into `artifacts.json` file in current working directory.